lib/rules/no-ember-super-in-es-classes.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       ) {
         context.report({
           node,
-          message: "Don't use `this._super` in ES classes",
+          message: "Don't use `this._super` in ES classes; instead, you should use super()",
```

Comment:
Can you put `super()` inside backticks for consistency?

Reply:
updated